spec/requests/sessions_spec.rb

```diff
@@ -10,5 +10,30 @@
         expect(response).to have_http_status(:ok)
       end
     end
+
+    it "does not extend an expired public user session" do
+      start_student_loans_claim
+
+      travel(2.hours) do
+        get refresh_session_path
```

Comment:
Since the response from this endpoint ([namely, whether a "success" or not](https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/master/app/assets/javascripts/components/timeout_dialog.js.erb), which I _think_ is derived from the response's status) determines the behaviour of the client-side code which makes the request, would we like to make an assertion about the response's status here? Same in the other example.

Reply:
I don't think the response is important ant enough to assert on here because this is very much an edge case and we wouldn't expect a user to make a request to the refresh endpoint under normal circumstances. The important thing is to be confident an exception does't get raised and cause noise in our monitoring (which is the underlying issue being resolved) and that the session is still expired.

Reply:
👍